### PR TITLE
Replace questions sidepanel

### PR DESCRIPTION
**Pull request by AllanOXDi**
## Summary
This PR was separated from #11314 to simplify testing and speedup the development process.

Note : This should not be merged before #11314

## References

#11016

## Reviewer guidance
To ensure easy testing of this future, please test #11314 first then come back to this.
While following this [figma design](https://www.figma.com/file/wRlClsHgcVqIwgUrntRN1a/Kolibri-Enhanced-Quiz-Management?type=design&node-id=28-12323&mode=design&t=YMS2eVHLdONzGCju-4)

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by MisRob** [comment]
@AllanOXDi 
> To ensure easy testing of this future, please test https://github.com/learningequality/kolibri/pull/11314 first then come back to this.

Does this mean we should wait for review here until feedback in the aforementioned PR is addressed and it's merged?

**Comment by AllanOXDi** [comment]
> @AllanOXDi
> 
> > To ensure easy testing of this future, please test #11314 first then come back to this.
> 
> Does this mean we should wait for review here until feedback in the aforementioned PR is addressed and it's merged?

Yes! I think that would be ideal .

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue [hunk 1]
```diff
@@ -1,21 +1,415 @@
...
+  }
+  .text-vertical-spacing {
+    margin-top: 0.5em;
+  }
+  .float-item-left-style {
+    float: right;
+    margin-top: 1em;
+  }
+  .choose-answer-style{
+    background-color:#FAFAFA;
```
Comment (MisRob) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue:370]: We don't hardcode colors into Kolibri (except a very few cases due to technical limitations). Tokens are used instead. This applies to other places with colors in this PR. Please see https://design-system.learningequality.org/colors/

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/AccordionQuizAnswer.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <div :class="isSelected ? 'quiz-answer-row-active' : 'quiz-answer-row' ">
+    <KGrid class="row-spacing">
+      <KGridItem
+        :layout12="{ span: 1 }"
+        :layout8="{ span: 1 }"
+      >
+        <div :class="isSelected ? 'circular-border-active' : 'circular-border'">
```
Comment (MisRob) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/AccordionQuizAnswer.vue:9]: How will we indicate that something is selected to screen readers?

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue [hunk 3]
```diff
@@ -1,21 +1,415 @@
...
+      </template>
+    </AccordionContainer>
+
+    <hr
+      class="horizontal-border"
+      style="margin-top:100px;width:100%;"
+    >
+    <div class="bottom-bar-style">
+      <KButton
+        text="replace"
```
Comment (MisRob) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue:221]: Seems like a missing translation string?

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue [hunk 4]
```diff
@@ -1,21 +1,415 @@
...
+          'Letters and Sounds Practice',
+          'Mathematics and Geometry',
+          'Algebra Practice',
+          'Long and Short Vowel Patterns: VCV and VCC Practice',
+        ],
+      };
+    },
+    $trs: {
+      replaceQuestionText: {
+        message: 'Replace questions',
```
Comment (MisRob) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue:293]: I thought we will be using the common translations approach for all new features, is there consensus about it? @marcellamaki 

It's described here in more detail https://kolibri-dev.readthedocs.io/en/develop/i18n.html#common-string-modules

**Comment by MisRob** [comment]
Thanks @AllanOXDi, lots of work here!

Due to the upcoming absences, I won't be able to do a complete review in a reasonable timeframe, apologies. If it's still valid the week after next, let me know and I can jump back in. At least, I'm leaving a few notes from a very brief skim through.

**Comment by marcellamaki** [comment]
Thanks @MisRob. @AllanOXDi - I think some of Misha's point have already been addressed by your work in https://github.com/learningequality/kolibri/pull/11314 and rebasing here may resolve some of that feedback. Let me know if you want some help with that. Otherwise, let me know when you have updated, and I'll be happy to do another review since Misha will be out for some days next week. 🙂

**Pull request closed without merging**